mreg_cli/api/models.py

```diff
@@ -1598,7 +1598,11 @@ def disassociate_mac(self) -> IPAddress:
 
         :returns: A new IPAddress object fetched from the API with the MAC address removed.
         """
-        return self.patch(fields={"macaddress": ""})
+        # Model converts empty string to None so we must validate this ourselves.
+        patched = self.patch(fields={"macaddress": ""}, validate=False)
```

Comment:
Should look into having `""` equal `None` when validating perhaps, but I fear that could create some hard to track down bugs unless we are certain this behavior is consistent in the API (which seems unlikely?!).